docker/Dockerfile

```diff
@@ -27,5 +12,19 @@ LABEL org.label-schema.vcs-url="https://github.com/openebs/cstor"
 LABEL org.label-schema.schema-version="1.0"
 LABEL org.label-schema.build-date=$BUILD_DATE
 
-ENTRYPOINT entrypoint-istgtimage.sh
 EXPOSE 3260 6060
+
```

Comment:
can we move this 'COPY'ing files section after apt-get install?
I'm thinking whether any package installation with 'apt-get install' may overwrite it. So, thinking that it would safe to put after all the packages installation?

Reply:
Sure enough. Maybe it is a safer approach.

Reply:
Resolved in e7a6cadba1e8b4eb1f302e47ba982a4853e26ad7